packages/client/src/components/RatingStars/RatingStars.jsx

```diff
@@ -28,8 +29,9 @@ export const RatingStars = ({
         name="half-rating"
         defaultValue={2.5}
         precision={0.5}
-        value={isNaN(rating) ? 0 : `${Number(rating)}`}
-        onChange={(e, v) => handleOpenReview(e, v)}
+        value={isNaN(rating) ? 0 : Number(rating)}
+        onClick={handleOpenReview ? (e, v) => handleOpenReview(e, v) : null}
```

Comment:
The check is here, because we reuse the component and in one of the usage scenarios it is "static" and should not react when clicked. So nothing is passed as a callback function. Consequently, the prop is undefined and throws a console error when the component is clicked without callback.
So we decided to bill it for this scenario to avoid error.
Alternatively, we could pass a dummy callback function like ()=>{}; but we didn't like that option...